sci-geosciences/gnome-maps/gnome-maps-43.0.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+inherit gnome.org gnome2-utils meson python-any-r1 xdg
+
+DESCRIPTION="A map application for GNOME"
+HOMEPAGE="https://wiki.gnome.org/Apps/Maps"
+
+LICENSE="GPL-2+ LGPL-2+ MIT CC-BY-3.0 CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+
+# Pure introspection dependencies found by grepping imports.gi in ${S}
+RDEPEND="
+	>=dev-libs/glib-2.66.0:2
+	x11-libs/pango[introspection]
+	>=dev-libs/gjs-1.69.2
+	>=dev-libs/gobject-introspection-0.10.1:=
+	gui-libs/gtk:4[introspection]
+	>=app-misc/geoclue-0.12.99:2.0[introspection]
+	>=gui-libs/libadwaita-1.0:1
+	>=sci-geosciences/geocode-glib-3.15.2[introspection]
```

Comment:
The `pkgdiff -b` output is

````diff
+geocodeglib = dependency('geocode-glib-2.0', version: '>= 3.15.2')
````

geocode-glib-2.0 is installed by geocode-glib only when configured with `-Dsoup2=false`, so we'll have to provide a slotted `sci-geosciences/geocode-glib` for libsoup2 vs libsoup3.